### PR TITLE
[Promote Rule] Potential Invoke-Mimikatz PowerShell Script

### PR DESCRIPTION
**Pull request by w0rk3r**
## Summary

Update the `Mimikatz Powershell Module Activity` rule to use PowerShell Script block logging and promotes it to production

image

### CODE REVIEW

File: rules/windows/credential_access_mimikatz_powershell_module.toml [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
-other features that make it useful for testing the security of networks. This rule detects the Invoke-Mimikatz
-PowerShell command.
+other features that make it useful for testing the security of networks. This rule detects Invoke-Mimikatz PowerShell
+script and alike.
 """
 from = "now-9m"
-index = ["logs-endpoint.events.*", "winlogbeat-*", "logs-windows.*"]
-language = "eql"
+index = ["winlogbeat-*", "logs-windows.*"]
+language = "kuery"
```
Comment (w0rk3r) [inline rules/windows/credential_access_mimikatz_powershell_module.toml:15]: This rule type change should not be a problem as Development rules are not included on the package, AFAIK.

File: rules/windows/credential_access_mimikatz_powershell_module.toml [hunk 2]
```diff
@@ -71,19 +77,43 @@ mean time to respond (MTTR).
...
 query = '''
-process where event.type in ("start", "process_started") and process.name : ("cmd.exe", "powershell.exe", "pwsh.exe")
-and process.args : ("*DumpCreds", "*Mimikatz*")
+event.category:process and
+powershell.file.script_block_text:(
+  (DumpCreds and
+  DumpCerts) or
+  "sekurlsa::logonpasswords" or
+  ("crypto::certificates" and
+  "CERT_SYSTEM_STORE_LOCAL_MACHINE")
```
Comment (terrancedejesus) [inline rules/windows/credential_access_mimikatz_powershell_module.toml:115]: Thoughts on adding more invokes for mimikatz?

```
CRYPTO::Certificates
KERBEROS::Golden
KERBEROS::List 
KERBEROS::PTT
LSADUMP::DCSync
LSADUMP::LSA
LSADUMP::SAM
LSADUMP::Trust
MISC::AddSid
MISC::MemSSP 
MISC::Skeleton
PRIVILEGE::Debug
SEKURLSA::Ekeys
SEKURLSA::Kerberos
SEKURLSA::Krbtgt
SEKURLSA::LogonPasswords
SEKURLSA::Pth
SEKURLSA::Tickets
TOKEN::List 
TOKEN::Elevate
TOKEN::Elevate /domainadmin 
```

Comment (w0rk3r) [reply rules/windows/credential_access_mimikatz_powershell_module.toml:115]: On Invoke-Mimikatz (https://raw.githubusercontent.com/EmpireProject/Empire/master/data/module_source/credentials/Invoke-Mimikatz.ps1), these are the only occurrences of hardcoded mimikatz commands (in the code, the privilege::debug is mentioned in the description), other commands can be passed as arguments, but they will not be captured in Powershell logs, only on command line ones.

Comment (terrancedejesus) [reply rules/windows/credential_access_mimikatz_powershell_module.toml:115]: Gotcha, thanks for explaining!

File: rules/windows/credential_access_mimikatz_powershell_module.toml (hunk 1 above)
Comment (brokensound77) [reply rules/windows/credential_access_mimikatz_powershell_module.toml:15]: yup - as long as it was never locked (released)

**Review by brokensound77 — APPROVED** [review]

**Pull request merged**